test/extended/util/annotate/rules.go

```diff
@@ -37,12 +37,19 @@ var (
 
 			// https://bugzilla.redhat.com/show_bug.cgi?id=1908645
 			`\[sig-network\] Networking Granular Checks: Services should function for service endpoints using hostNetwork`,
+			`\[sig-network\] Networking Granular Checks: Services should function for pod-Service\(hostNetwork\)`,
 
-			// https://bugzilla.redhat.com/show_bug.cgi?id=1945085
-			`API data in etcd should be stored at the correct location and version for all resources`,
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1952460
+			`\[sig-network\] Firewall rule control plane should not expose well-known ports`,
 
-			// // https://bugzilla.redhat.com/show_bug.cgi?id=1945091
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1952457
+			`\[sig-node\] crictl should be able to run crictl on the node`,
```

Comment:
Can we enable this test again to gather the failure results?

Reply:
@saschagrunert I'd suggest opening a separate PR, I'm trying to land this for a while now :sweat_smile:

Reply:
Sounds good to me, thank you! :)